package.json

```diff
@@ -12,46 +12,41 @@
     "format": "prettier --write ."
   },
   "dependencies": {
-    "@chakra-ui/icons": "^1.1.1",
-    "@chakra-ui/react": "^1.7.4",
-    "@emotion/core": "^11.0.0",
-    "@emotion/react": "^11.4.1",
-    "@emotion/styled": "^11.3.0",
+    "@chakra-ui/icons": "^2.0.19",
+    "@chakra-ui/react": "^2.6.1",
+    "@emotion/react": "^11.11.0",
+    "@emotion/styled": "^11.11.0",
     "@fontsource/inter": "^4.5.15",
     "@hookform/resolvers": "^2.8.8",
-    "@react-google-maps/api": "^2.7.0",
+    "@react-google-maps/api": "^2.18.1",
+    "@tanstack/react-query": "^4.29.5",
     "airtable": "^0.11.3",
     "axios": "^0.26.0",
-    "emotion-theming": "^11.0.0",
     "firebase": "^9.10.0",
-    "framer-motion": "^4.1.17",
+    "framer-motion": "^10.12.9",
     "fuse.js": "^6.6.2",
     "geolib": "^3.3.3",
-    "next": "^11.1.4",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "next": "^13.3.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "react-feather": "^2.0.10",
-    "react-hook-form": "^7.26.0",
-    "react-icons": "^4.3.1",
-    "react-paginated-list": "^1.1.5",
-    "react-query": "^3.28.0",
-    "react-select": "^5.2.1",
-    "styled-components": "^5.3.3",
```

Comment:
@notnazsty Yarn says `styled-components` is a peer dependency of `react-paginated-list` - lmk if I should add it back.

Also, now that we have our own hook + component for pagination, we should refactor out `react-paginated-list` (rn it's used on the `[listId].tsx` page.

Reply:
Seems like it's needed for now 😬 

<img width="962" alt="Screenshot 2023-05-09 at 8 24 25 PM" src="https://github.com/MutualAidNYC/services-lists/assets/54749942/00bfc52c-bdfd-411c-bb3b-1696f2db759d">